src/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "short_name": "mkdocs-material",
+  "name": "mkdocs-material",
+  "start_url": "/?utm_source=web_app_manifest",
+  "icons": [
+    {
+      "src": "assets/images/favicon.png",
```

Comment:
would love to use mkdocs.yml parameters here but not sure how.

Reply:
I think that's not possible, since Jinja only accepts `.html` and not `.json` files.